config/src/config/quorum_store_config.rs

```diff
@@ -39,23 +63,21 @@ impl Default for QuorumStoreConfig {
             channel_size: 1000,
             proof_timeout_ms: 10000,
             batch_request_num_peers: 2,
-            mempool_pulling_interval: 100,
-            end_batch_ms: 500,
-            max_batch_counts: 300,
-            max_batch_bytes: 1000000,
+            mempool_pulling_interval: 250,
+            // TODO: This essentially turns fragments off, because there was performance degradation. Needs more investigation.
+            end_batch_ms: 10,
+            max_batch_bytes: 4 * 1024 * 1024,
             batch_request_timeout_ms: 10000,
             batch_expiry_round_gap_when_init: 100,
             batch_expiry_round_gap_behind_latest_certified: 500,
             batch_expiry_round_gap_beyond_latest_certified: 500,
             batch_expiry_grace_rounds: 5,
             memory_quota: 100000000,
             db_quota: 10000000000,
-            mempool_txn_pull_max_count: 300,
-            mempool_txn_pull_max_bytes: 1000000,
-            // QS will be backpressured if the remaining local batches is more than this number
-            back_pressure_local_batch_num: 10,
+            mempool_txn_pull_max_bytes: 4 * 1024 * 1024,
+            back_pressure: QuorumStoreBackPressureConfig::default(),
             // number of batch coordinators to handle QS Fragment messages, should be >= 1
-            num_workers_for_remote_fragments: 2,
+            num_workers_for_remote_fragments: 10,
```

Comment:
why this change?

Reply:
This was likely touched when we were doing larger cluster tests, but don't remember how significant this was.

We need to do 400 PoS signatures per second. I'll keep this for now, but we can follow-up with more tuning.